pkg/cephmgr/rgw/leader.go

```diff
@@ -163,6 +163,24 @@ func markUnapplied(context *clusterd.Context) error {
 }
 
 func (r *Leader) enable(context *clusterd.Context, factory client.ConnectionFactory) error {
```

Comment:
who calls RGW enable in this new world? i thought the operator just creates a RGW daemon and starts it